backend/cfg/cfg_regalloc_utils.ml

```diff
@@ -3,6 +3,11 @@
 module Array = ArrayLabels
 module List = ListLabels
 
+let bool_of_env env_var =
+  match Sys.getenv_opt env_var with Some "1" -> true | Some _ | None -> false
+
+let regalloc_debug = bool_of_env "REGALLOC_DEBUG"
```

Comment:
Changed env variable to `CFG_REGALLOC_VALIDATOR_DEBUG`.
Changed ocaml name to `validator_debug` because `cfg_regalloc` is already in module name `cfg_regalloc_utils`